test/src/test_reader.cpp

```diff
@@ -3,7 +3,8 @@
 //
 // Distributed under the "BSD License". See the accompanying LICENSE.rst file
 
-#include <bitter/reader.hpp>
+#include <bitter/lsb0_reader.hpp>
+#include <bitter/msb0_reader.hpp>
```

Comment:
I would prefer to have two separate test files for the two readers (if they are to be split like this, that is).